l10n/en/products/vpn/more/vpn-or-proxy.ftl

```diff
@@ -0,0 +1,58 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/vpn/more/vpn-or-proxy/
+
+# HTML page title and main title
+vpn-or-proxy-the-difference-between = The difference between a VPN and a web proxy
+
+# HTML page description and intro
+vpn-or-proxy-vpns-and-proxies-are = VPNs and proxies are solutions for online privacy and security. Here’s how these protect you and how to choose the best option.
+
+vpn-or-proxy-virtual-private-networks = Virtual private networks (VPNs) and secure web proxies are solutions for better privacy and security online, but it can be confusing to figure out which one is right for you. Here’s a look at how these services protect you and how to choose the best option for when you’re online.
+vpn-or-proxy-stop-isps-from = Stop ISPs from spying on you
+
+# Variables
+#   $firefox (url) - https://www.mozilla.org/firefox/new/
+#   $etp (url) - https://blog.mozilla.org/products/firefox/control-trackers-with-firefox/
+#   $trackers (url) - https://blog.mozilla.org/internet-culture/mozilla-explains/what-is-a-web-tracker/
+vpn-or-proxy-when-you-use-firefox = When you <a href="{ $firefox }">use { -brand-name-firefox }</a>, <a href="{ $etp }">enhanced tracking protection</a> automatically blocks many third party <a href="{ $trackers }">web trackers</a> from following you around the web. But here’s an interesting fact: your internet service provider (ISP) that you are paying for an internet connection — can still observe and track you.
```

Comment:
Should `enhanced tracking protection` be capitalized since it's a product feature?
```suggestion
vpn-or-proxy-when-you-use-firefox = When you <a href="{ $firefox }">use { -brand-name-firefox }</a>, <a href="{ $etp }">Enhanced Tracking Protection</a> automatically blocks many third party <a href="{ $trackers }">web trackers</a> from following you around the web. But here’s an interesting fact: your internet service provider (ISP) that you are paying for an internet connection — can still observe and track you.
```